grc/gui/MainWindow.py

```diff
@@ -58,18 +59,29 @@ def __init__(self, app, platform, action_handler_callback):
         log.debug("__init__()")
 
         self._platform = platform
+        self.app = app
         self.config = platform.config
 
+        # Add all "win" actions to the local
+        win_actions = filter(lambda x: x.startswith("win."), Actions.get_actions())
+        map(lambda x: self.add_action(Actions.actions[x]), win_actions)
```

Comment:
map and lambda, Guido van Rossum would hate you (: 
@skoslowski wouldn't a list comprehension work as well, or is that what you meant?